lib/src/gutter.dart

```diff
@@ -1,54 +1,123 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
 
-import 'package:flutter_gutter/flutter_gutter.dart';
-
-/// A gap twice the standard gutter size according to Material Design's
-/// breakpoints system.
-class GutterLarge extends StatelessWidget {
-  const GutterLarge({super.key});
-
-  @override
-  Widget build(BuildContext context) => Gap(size: context.gutterLarge);
-}
+import '../flutter_gutter.dart';
 
 /// A gap of the standard gutter size according to Material Design's breakpoints
 /// system.
+/// Creates a `SizedBox` with a height of size corresponding to the layout axis
+/// of the first `Scrollable`, `Row` or `Column` above this widget.
 class Gutter extends StatelessWidget {
-  const Gutter({super.key});
+  /// Creates a new [Gutter] widget.
+  const Gutter({
```

Comment:
I actually really like this change where `Gutter` now behaves as the base widget and includes size overrides. I think in the old version `Gap` was not very discoverable, nice work!